src/protocol/ipa/mod.rs

```diff
@@ -302,6 +302,7 @@ pub mod tests {
 
     #[tokio::test]
     #[allow(clippy::missing_panics_doc)]
+    #[ignore]
```

Comment:
How about changing batchsize to 10 or even 5 if needed to keep it running?

Reply:
I like this test because it shows us the end-to-end performance we can use to see our progress in optimizations. It won't serve the purpose if we decrease the batch size to 10 or 5, and certainly we don't need to run it as an unit test.